config/config.go

```diff
@@ -71,6 +72,12 @@ type Hypervisor struct {
 	OvmControl string
 }
 
+// ExadataFeature holds exadata feature params
+type ExadataFeature struct {
+	Enabled     bool
+	NotRootUser string
```

Comment:
Can you consider to create a CommonFeature struct that contains the Enabled and NotRootUser field. Is NotRootUser a good name? Maybe OverrideUser is a better name?

Reply:
But FetchersUser is used only by ExadataFeature for now!